users/views.py

```
@@ -1004,3 +1004,11 @@ def get(self, request, *args, **kwargs):
         session_phone_device.session.save()
 
         return JsonResponse({"otp": session_phone_device.token})
+
+
+class FetchUserAnalytics(ClientProtectedResourceMixin, View):
+    required_scopes = ["user_fetch"]
```

Ruff (0.14.5)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)